frontend/src/pages/image/collection.vue

```diff
@@ -1,5 +1,9 @@
 <template>
-  <div class="p-6 pt-0 lg:p-10 lg:pt-2">
+  <div
+    :id="`#${skipToContentTargetId}`"
```

Comment:
```suggestion
    :id="skipToContentTargetId"
```

This fixes the skip to content link on the collection pages :smile: